src/codegen/Common.ts

```diff
@@ -144,6 +144,13 @@ const common = {
         'Describes the location in the data model where the component should store its value(s). A simple ' +
           'binding is used for components that only store a single value, usually a string.',
       ),
+  IDataModelBindingsOptionsSimple: () =>
+    new CG.obj(new CG.prop('simpleBinding', new CG.str()), new CG.prop('metadata', new CG.str().optional()))
```

Comment:
There should probably also be a description for the `metadata` prop here as well, although I'm not sure if it's easily visible from vscode that way. I know people will wonder what it's for without any docs! 😊

Reply:
I'll add some description! 😁